features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/ResourceResolver.kt

```diff
@@ -127,6 +163,36 @@ internal class ResourceResolver(
         }
     }
 
+    @WorkerThread
+    private fun resolveBitmapHash(
+        bitmap: Bitmap,
+        compressedBitmapBytes: ByteArray,
+        shouldCacheBitmap: Boolean,
+        resolveResourceCallback: ResolveResourceCallback
+    ) {
+        // failed to get image data
+        if (compressedBitmapBytes.isEmpty()) {
+            // we are already logging the failure in webpImageCompression
+            resolveResourceCallback.onFailed()
+            return
+        }
+
+        val resourceId = md5HashGenerator.generate(compressedBitmapBytes)
+
+        // failed to resolve bitmap identifier
+        if (resourceId == null) {
+            // logging md5 generation failures inside md5HashGenerator
+            resolveResourceCallback.onFailed()
+            return
+        }
+
+        if (shouldCacheBitmap) {
+            bitmapCachesManager.putInBitmapPool(bitmap)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏ Why are we caching the bitmap here ? Shouldn't we use this cached image somewhere ?

Reply:
after discussed with @jonathanmos , we decided to remove the cache in this PR, and do this later